src/datagrid/DataGrid.tsx

```diff
@@ -127,9 +127,11 @@ export type DataGridFooter = {
  * type for DataGridSort :
  * @param {defaultSortOrder} if data in column by default sorted
  * @param {sortFunction} function to perform sorting
+ * @param {isCurrentlySorted} checks if column is currently sorted or not
  */
 export type DataGridSort = {
     defaultSortOrder: SortOrder;
+    isCurrentlySorted?: boolean;
```

Comment:
please change isCurrentlySorted to isSorted ? looks cleaner.

Reply:
done